main.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+$(document).ready(function(){
+
+    //cheese pizza calculations here
+
+    let cheeseCalorieCounter = +$('.calorie_num').val();
```

Comment:
I haven't brushed up on JQuery in a while - what does '+' before the reference $ mean? For my own curiosity :)

Reply:
think it makes it a number

Reply:
That was so that it would increment that value of the calorie num for every button click : -)